GalaxyView.cpp

```diff
@@ -290,7 +369,7 @@ void GalaxyView::RandomizeCommodity()
                 for(const System *source : sources)
                     for(const QString &name : source->Links())
                     {
-                        auto it = mapData.Systems().find(name);
+                        const auto &it = mapData.Systems().find(name);
```

Comment:
Since `find()` is returning an iterator by value (specifically a `map<QString, System>::iterator`), it doesn't make sense for this to be a reference. It ends up being a reference to a temporary value, which is legal C++ but not entirely kosher.